### PR TITLE
fix/AB#77318_resources-records-pagination

### PR DESCRIPTION
**Pull request by matheus-relief**
# Description

This PR fixes the pagination in the resource records. The issue was we were creating new instances of the paginator every time we loaded new data, that would cause it to loose its current state. Locally I couldn't reproduce double actions columns.

## Useful links

- Please insert [link to ticket](https://dev.azure.com/WHOHQ/EMSSAFE/_workitems/edit/77318)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Improvement (refactor or addition to existing functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# How Has This Been Tested?

See below

## Screenshots
Before:
Peek 2023-08-30 03-01

After:
Peek 2023-08-30 03-01

# Checklist:

( * == Mandatory ) 

- [x] * I have set myself as assignee of the pull request
- [x] * My code follows the style guidelines of this project
- [x] * Linting does not generate new warnings
- [x] * I have performed a self-review of my own code
- [x] * I have put the ticket for review, adding the oort-frontend team to the list of reviewers
- [x] * I have commented my code, particularly in hard-to-understand areas
- [x] * I have put JSDoc comment in all required places
- [x] * My changes generate no new warnings
- [x] * I have included screenshots describing my changes if relevant
- [x] * I have selected labels in the Pull Request, according to the changes with code brings
- [ ] I have made corresponding changes to the documentation ( if required )
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by unai-reliefapp** [comment]
Hi @matheus-relief , can you please double check rest of the paginated tables in the application please?
I see that for example in the [pull-jobs.html](https://github.com/ReliefApplications/ems-frontend/blob/beta/apps/back-office/src/app/dashboard/pages/pull-jobs/pull-jobs.component.html#L98-L106) file we have the same element order and we have the same problem

PS: If you are working on WHO this week, if not let me know and I'll push the changes

Thanks!

**Comment by matheus-relief** [comment]
> Hi @matheus-relief , can you please double check rest of the paginated tables in the application please? I see that for example in the [pull-jobs.html](https://github.com/ReliefApplications/ems-frontend/blob/beta/apps/back-office/src/app/dashboard/pages/pull-jobs/pull-jobs.component.html#L98-L106) file we have the same element order and we have the same problem
> 
> PS: If you are working on WHO this week, if not let me know and I'll push the changes
> 
> Thanks!

Done @unai-reliefapp, the only other I could find that had a problem with pagination was the custom notifications one, but that's not being used for now, so instead I just hid the page in the menu.

**Review by RenzoPrats — APPROVED** [review]
Working fine, looks good to me!

**Review by unai-reliefapp — APPROVED** [review]

File: apps/back-office/src/app/dashboard/pages/resource/records-tab/records-tab.component.html [hunk 1]
```diff
@@ -180,6 +176,14 @@
     ></shared-empty>
   </ng-template>
 </ng-container>
+<!-- Pagination -->
+<ui-paginator
```
Comment (AntoineRelief) [inline apps/back-office/src/app/dashboard/pages/resource/records-tab/records-tab.component.html:180]: @matheus-relief 
cc @unai-reliefapp 
please note that it does not prevent this to happen:
Capture d’écran 2023-10-17 à 11 19 11

I'm fixing it in a new commit

Comment (unai-reliefapp) [reply apps/back-office/src/app/dashboard/pages/resource/records-tab/records-tab.component.html:180]: noted, thanks Antoine

**Pull request merged**